### ISSUE TITLE
Server Data Error I Don't Understand

### ISSUE THREAD

**Issue by LordBevis** [BlackJar72/DoomlikeDungeons#43]
I am running hexxit updated mod pack by Cricrane from curseforge its the new years edition of the mod pack.
I got this server error and I have tried alot of stuff.
Still dont really understand whats wrong with it.
At first It seemed like it was a Json issue but I had the server regenerate that file and it appeared the same as before so I just dont think thats it.
This issue is really confusing to me.
I tried removing the mod jar file that did not work.
I also tried getting a newer version of the mod jar file to replace that one and I got the same exact error.
This all makes even less sense because I got the data for this modded server from a friend who ran it.
Last time we ran this server it worked so I don't really understand how this problem could have formed we have touched nothing that I can think of between now and then that would have caused something like this. It should just work from the get go as I start it up the same as it worked about 3 years ago.
I googled the problem. Some were saying it was a RAM amount issue but thats not the case as I have plenty of RAM and have also allocated it in the launch.bat file.
Some were saying it was a problem with the mod itself and to reinstall it but seeing as how I tried a newer version of the mod and got the exact same problem I just dont think thats the case. I also tried ChatGPT this is how I came to think it was a Json file issue but I really just dont think thats the problem because the file was regenerated like I said. Can someone tell me what is wrong? the crash log is big and harder for me to interpret as I just dont have alot of experience with that sort of thing. Despite having the crash log this has turned into a wild goose chase. Here is the crash log below. Thanks in advance to anyone who can help me out.

---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 2024-10-01 14:55:36 PDT
Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Doomlike Dungeons (dldungeonsjbg)
Caused by: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 14 path $
    at com.google.gson.JsonParser.parse(JsonParser.java:65)
    at com.google.gson.JsonParser.parse(JsonParser.java:45)
    at jaredbgreat.dldungeons.builder.BlockFamily.makeBlockFamily(BlockFamily.java:80)
    at jaredbgreat.dldungeons.themes.ThemeReader.readBlockFamilies(ThemeReader.java:266)
    at jaredbgreat.dldungeons.themes.ThemeReader.findBlockFiles(ThemeReader.java:198)
    at jaredbgreat.dldungeons.themes.ThemeReader.readThemes(ThemeReader.java:227)
    at jaredbgreat.dldungeons.DoomlikeDungeons.postInit(DoomlikeDungeons.java:63)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
    at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
    at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:754)
    at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:108)
    at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:338)
    at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:219)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
    at java.lang.Thread.run(Unknown Source)
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 14 path $
    at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1559)
    at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1401)
    at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:542)
    at com.google.gson.stream.JsonReader.peek(JsonReader.java:425)
    at com.google.gson.JsonParser.parse(JsonParser.java:60)
    ... 41 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 10 (amd64) version 10.0
  Java Version: 1.8.0_231, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 3233808360 bytes (3083 MB) / 6442450944 bytes (6144 MB) up to 6442450944 bytes (6144 MB)
  JVM Flags: 3 total; -Xms6G -Xmx6G -XX:+UseG1GC
  IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
  FML: MCP 9.42 Powered by Forge 14.23.5.2854 88 mods loaded, 88 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State | ID                  | Version                  | Source                                         | Signature                                |
       |:----- |:------------------- |:------------------------ |:---------------------------------------------- |:---------------------------------------- |
       | LCHIJ | minecraft           | 1.12.2                   | minecraft.jar                                  | None                                     |
       | LCHIJ | mcp                 | 9.42                     | minecraft.jar                                  | None                                     |
       | LCHIJ | FML                 | 8.0.99.99                | forge-1.12.2-14.23.5.2854.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJ | forge               | 14.23.5.2854             | forge-1.12.2-14.23.5.2854.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJ | ivtoolkit           | 1.3.3-1.12               | minecraft.jar                                  | None                                     |
       | LCHIJ | foamfixcore         | 7.7.4                    | minecraft.jar                                  | None                                     |
       | LCHIJ | srm-hooks           | 1.12.2-1.0.0             | minecraft.jar                                  | None                                     |
       | LCHIJ | randompatches       | 1.12.2-1.22.1.10         | randompatches-1.12.2-1.22.1.10.jar             | None                                     |
       | LCHIJ | jei                 | 4.16.1.301               | jei_1.12.2-4.16.1.301.jar                      | None                                     |
       | LCHIJ | abyssalcraft        | 1.10.3                   | AbyssalCraft-1.12.2-1.10.3.jar                 | 220f10d3a93b3ff5fbaa7434cc629d863d6751b9 |
       | LCHIJ | chisel              | MC1.12.2-1.0.2.45        | Chisel-MC1.12.2-1.0.2.45.jar                   | None                                     |
       | LCHIJ | mantle              | 1.12-1.3.3.55            | Mantle-1.12-1.3.3.55.jar                       | None                                     |
       | LCHIJ | quark               | r1.6-179                 | Quark-r1.6-179.jar                             | None                                     |
       | LCHIJ | twilightforest      | 3.11.1021                | twilightforest-1.12.2-3.11.1021-universal.jar  | None                                     |
       | LCHIJ | tconstruct          | 1.12.2-2.13.0.183        | TConstruct-1.12.2-2.13.0.183.jar               | None                                     |
       | LCHIJ | acintegration       | 1.11.2                   | AbyssalCraft Integration-1.12.2-1.11.2.jar     | 220f10d3a93b3ff5fbaa7434cc629d863d6751b9 |
       | LCHIJ | aether_legacy       | 1.5.3.1                  | aether-1.12.2-v1.5.3.1.jar                     | None                                     |
       | LCHIJ | aether_legacy_addon | 1.12.2-v1.3.0            | Aether Continuation v1.3.0.jar                 | None                                     |
       | LCHIJ | aireducer           | 0.2.1                    | AIReducer-1.12.2-0.2.1.jar                     | None                                     |
       | LCHIJ | aoa3                | 3.3.6                    | AoA3-3.3.6.jar                                 | 8bddeac148843f40e0b9f69201be2f4b3071700d |
       | LCHIJ | appleskin           | 1.0.14                   | AppleSkin-mc1.12-1.0.14.jar                    | None                                     |
       | LCHIJ | attributefix        | 1.0.4                    | AttributeFix-1.12.2-1.0.4.jar                  | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJ | atum                | 2.0.20                   | Atum-1.12.2-2.0.20.jar                         | None                                     |
       | LCHIJ | autoreglib          | 1.3-32                   | AutoRegLib-1.3-32.jar                          | None                                     |
       | LCHIJ | battletowers        | 1.6.5                    | BattleTowers-1.12.2.jar                        | None                                     |
       | LCHIJ | bibliocraft         | 2.4.5                    | BiblioCraft[v2.4.5][MC1.12.2].jar              | None                                     |
       | LCHIJ | biomesoplenty       | 7.0.1.2441               | BiomesOPlenty-1.12.2-7.0.1.2441-universal.jar  | None                                     |
       | LCHIJ | bookshelf           | 2.3.590                  | Bookshelf-1.12.2-2.3.590.jar                   | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJ | brokenwings         | 2.0.0                    | brokenwings-2.1.1.jar                          | None                                     |
       | LCHIJ | chameleon           | 1.12-4.1.3               | Chameleon-1.12-4.1.3.jar                       | None                                     |
       | LCHIJ | chiselsandbits      | 14.33                    | chiselsandbits-14.33.jar                       | None                                     |
       | LCHIJ | chococraft          | 0.9.9.0                  | chococraft_1.12.2-0.9.9.0.jar                  | None                                     |
       | LCHIJ | neid                | 1.5.4.4                  | NotEnoughIDs-1.5.4.4.jar                       | None                                     |
       | LCHIJ | phosphor-lighting   | 1.12.2-0.2.6             | phosphor-1.12.2-0.2.6+build50-universal.jar    | f0387d288626cc2d937daa504e74af570c52a2f1 |
       | LCHIJ | cqrepoured          | 2.2.4B                   | Chocolate_Quest_Repoured-1.12.2-2.2.4B.jar     | None                                     |
       | LCHIJ | conarm              | 1.2.5.10                 | conarm-1.12.2-1.2.5.10.jar                     | b33d2c8df492beff56d1bbbc92da49b8ab7345a1 |
       | LCHIJ | dimdoors            | 3.0.9+287                | DimensionalDoors-3.0.9-287.jar                 | None                                     |
       | LCHIE | dldungeonsjbg       | 1.14.12                  | DoomlikeDungeons-1.14.12-MC1.12.2.jar          | None                                     |
       | LCHI  | dungeonsmod         | 1.12.2-1.0.6             | DungeonsMod-1.12.2-1.0.6.jar                   | None                                     |
       | LCHI  | dungeontactics      | DT-0.16.9                | DungeonTactics-1.12.2-0.16.9.jar               | None                                     |
       | LCHI  | eplus               | 5.0.176                  | EnchantingPlus-1.12.2-5.0.176.jar              | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHI  | erebus              | 1.0.31                   | Erebus-1.0.31.jar                              | None                                     |
       | LCHI  | fastfurnace         | 1.3.1                    | FastFurnace-1.12.2-1.3.1.jar                   | None                                     |
       | LCHI  | fastbench           | 1.7.3                    | FastWorkbench-1.12.2-1.7.3.jar                 | None                                     |
       | LCHI  | foamfix             | 0.10.11-1.12.2           | foamfix-0.10.11-1.12.2.jar                     | None                                     |
       | LCHI  | futureminecraf      | 1.3.1                    | FutureVersions v1.3.1.jar                      | None                                     |
       | LCHI  | gottschcore         | 1.14.0                   | GottschCore-mc1.12.2-f14.23.5.2854-v1.14.0.jar | None                                     |
       | LCHI  | placebo             | 1.6.0                    | Placebo-1.12.2-1.6.0.jar                       | None                                     |
       | LCHI  | hexxitgear          | 2.8.0                    | Hexxit Gear-1.12.2-2.8.0.jar                   | None                                     |
       | LCHI  | infernalmobs        | 1.7.6                    | InfernalMobs-1.12.2.jar                        | None                                     |
       | LCHI  | inventorytweaks     | 1.63+release.109.220f184 | InventoryTweaks-1.63.jar                       | 55d2cd4f5f0961410bf7b91ef6c6bf00a766dcbe |
       | LCHI  | ironchest           | 1.12.2-7.0.67.844        | ironchest-1.12.2-7.0.72.847.jar                | None                                     |
       | LCHI  | journeymap          | 1.12.2-5.7.1             | journeymap-1.12.2-5.7.1.jar                    | None                                     |
       | LCHI  | jrftl               | 1.1                      | JRFTL[1.12.2]-1.1.jar                          | None                                     |
       | LCHI  | login_shield        | 1.12.2-6-g5654706        | Login_Shield-1.12.2-6-g5654706.jar             | None                                     |
       | LCHI  | lootbags            | 2.5.8.6                  | lootbags-2.5.8.6.jar                           | None                                     |
       | LCHI  | lost_aether         | 1.0.2                    | lost-aether-content-1.12.2-1.0.2.jar           | None                                     |
       | LCHI  | malisiscore         | 1.12.2-6.5.1-SNAPSHOT    | malisiscore-1.12.2-6.5.1.jar                   | None                                     |
       | LCHI  | malisisdoors        | 1.12.2-7.3.0             | malisisdoors-1.12.2-7.3.0.jar                  | None                                     |
       | LCHI  | malisisswitches     | 1.12.2-5.1.0             | malisisswitches-1.12.2-5.1.0.jar               | None                                     |
       | LCHI  | morebeautifulplates | 1.12.2-1.0.3             | morebeautifulplates-1.12.2-1.0.3.jar           | None                                     |
       | LCHI  | natura              | 1.12.2-4.3.2.69          | natura-1.12.2-4.3.2.69.jar                     | None                                     |
       | LCHI  | netherportalfix     | 5.3.17                   | NetherPortalFix_1.12.1-5.3.17.jar              | None                                     |
       | LCHI  | recipehandler       | 0.13                     | NoMoreRecipeConflict-0.13(1.12.2).jar          | None                                     |
       | LCHI  | oretweaker          | 1.0.2-1.12.0             | Ore Tweaker-1.0.2-1.12.0.jar                   | None                                     |
       | LCHI  | patchouli           | 1.0-23.6                 | Patchouli-1.0-23.6.jar                         | None                                     |
       | LCHI  | performant          | 1.12.2-1.4               | performant-1.12.2-1.10.jar                     | None                                     |
       | LCHI  | reccomplex          | 1.4.8.2                  | RecurrentComplex-1.4.8.2.jar                   | None                                     |
       | LCHI  | xreliquary          | 1.12.2-1.3.4.796         | Reliquary-1.12.2-1.3.4.796.jar                 | None                                     |
       | LCHI  | roguelike           | 2.3.1                    | RoguelikeDungeonsFnarEdition-1.12.2-2.3.1.jar  | None                                     |
       | LCHI  | roughmobsrevamped   | 2.3.21                   | roughmobsrevamped-1.12.2-2.3.21.jar            | None                                     |
       | LCHI  | secretroomsmod      | 5.6.4                    | secretroomsmod-1.12.2-5.6.4.jar                | None                                     |
       | LCHI  | soulshardsrespawn   | 1.12.2-1.1.1-13          | SoulShardsRespawn-1.12.2-1.1.1-13.jar          | None                                     |
       | LCHI  | spartanshields      | 1.5.4                    | SpartanShields-1.12.2-1.5.4.jar                | None                                     |
       | LCHI  | storagedrawers      | 5.2.2                    | StorageDrawers-1.12.2-5.4.2.jar                | None                                     |
       | LCHI  | surge               | 2.0.77                   | Surge-1.12.2-2.0.77.jar                        | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHI  | theoneprobe         | 1.4.28                   | theoneprobe-1.12-1.4.28.jar                    | None                                     |
       | LCHI  | tcomplement         | 1.12.2-0.4.3             | TinkersComplement-1.12.2-0.4.3.jar             | None                                     |
       | LCHI  | tombmanygraves      | 1.12-4.2.0               | TombManyGraves-1.12-4.2.0.jar                  | None                                     |
       | LCHI  | treechopper         | 1.2.4                    | TreeChopper-1.12.2-1.2.4.jar                   | None                                     |
       | LCHI  | unloader            | 1.2.0                    | unloader-1.2.0.jar                             | None                                     |
       | LCHI  | vanillafix          | 1.0.10-150               | VanillaFix-1.0.10-150.jar                      | None                                     |
       | LCHI  | waddles             | 0.6.0                    | Waddles-1.12.2-0.6.0.jar                       | None                                     |
       | LCHI  | wearablebackpacks   | 3.1.4                    | WearableBackpacks-1.12.2-3.1.4.jar             | None                                     |
       | LCHI  | worleycaves         | 1.5.2                    | worleycaves-1.12.2-1.5.2.jar                   | None                                     |
       | LCHI  | llibrary            | 1.7.20                   | llibrary-1.7.20-1.12.2.jar                     | b9f30a813bee3b9dd5652c460310cfcd54f6b7ec |
       | LCHI  | orelib              | 3.6.0.1                  | OreLib-1.12.2-3.6.0.1.jar                      | 7a2128d395ad96ceb9d9030fbd41d035b435753a |
       | LCHI  | thebetweenlands     | 3.7.2                    | TheBetweenlands-3.7.2-universal.jar            | 38067d6878811efb38b6a045521cfd80b9b60b38 |
  Loaded coremods (and transformers): MalisisSwitchesPlugin (malisisswitches-1.12.2-5.1.0.jar)
                                        
                                      RandomPatches (randompatches-1.12.2-1.22.1.10.jar)
                                        com.therandomlabs.randompatches.core.RPTransformer
                                      Quark Plugin (Quark-r1.6-179.jar)
                                        vazkii.quark.base.asm.ClassTransformer
                                      Do not report to Forge! (If you haven't disabled the FoamFix coremod, try disabling it in the config! Note that this bit of text will still appear.) (foamfix-0.10.11-1.12.2.jar)
                                        pl.asie.foamfix.coremod.FoamFixTransformer
                                      PerformantLoadingPlugin (performant-1.12.2-1.10.jar)
                                        
                                      Born in a Barn (Born In A Barn V1.8-1.12-1.1.jar)
                                        com.chocohead.biab.BornInABarn
                                      TheBetweenlandsLoadingPlugin (TheBetweenlands-3.7.2-core.jar)
                                        thebetweenlands.core.TheBetweenlandsClassTransformer
                                      Plugin (NotEnoughIDs-1.5.4.4.jar)
                                        ru.fewizz.neid.asm.Transformer
                                      llibrary (llibrary-core-1.0.11-1.12.2.jar)
                                        net.ilexiconn.llibrary.server.core.plugin.LLibraryTransformer
                                        net.ilexiconn.llibrary.server.core.patcher.LLibraryRuntimePatcher
                                      Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
                                        invtweaks.forge.asm.ContainerTransformer
                                      VanillaFixLoadingPlugin (VanillaFix-1.0.10-150.jar)
                                        
                                      PhosphorFMLLoadingPlugin (phosphor-1.12.2-0.2.6+build50-universal.jar)
                                        
                                      SecretRoomsMod-Core (secretroomsmod-1.12.2-5.6.4.jar)
                                        com.wynprice.secretroomsmod.core.SecretRoomsTransformer
                                      MalisisCorePlugin (malisiscore-1.12.2-6.5.1.jar)
                                        
                                      SurgeLoadingPlugin (Surge-1.12.2-2.0.77.jar)
                                        
                                      IvToolkit (IvToolkit-1.3.3-1.12.jar)
  Pulsar/tconstruct loaded Pulses: - TinkerCommons (Enabled/Forced)
                                   - TinkerWorld (Enabled/Not Forced)
                                   - TinkerTools (Enabled/Not Forced)
                                   - TinkerHarvestTools (Enabled/Forced)
                                   - TinkerMeleeWeapons (Enabled/Forced)
                                   - TinkerRangedWeapons (Enabled/Forced)
                                   - TinkerModifiers (Enabled/Forced)
                                   - TinkerSmeltery (Enabled/Not Forced)
                                   - TinkerGadgets (Enabled/Not Forced)
                                   - TinkerOredict (Enabled/Forced)
                                   - TinkerIntegration (Enabled/Forced)
                                   - TinkerFluids (Enabled/Forced)
                                   - TinkerMaterials (Enabled/Forced)
                                   - TinkerModelRegister (Enabled/Forced)
                                   - chiselIntegration (Enabled/Not Forced)
                                   - chiselsandbitsIntegration (Enabled/Not Forced)
                                   - theoneprobeIntegration (Enabled/Not Forced)
                                   - quarkIntegration (Enabled/Not Forced)
  Pulsar/natura loaded Pulses: - NaturaCommons (Enabled/Forced)
                               - NaturaOverworld (Enabled/Not Forced)
                               - NaturaNether (Enabled/Not Forced)
                               - NaturaDecorative (Enabled/Not Forced)
                               - NaturaTools (Enabled/Not Forced)
                               - NaturaEntities (Enabled/Not Forced)
                               - NaturaOredict (Enabled/Forced)
                               - NaturaWorld (Enabled/Not Forced)
  Pulsar/tcomplement loaded Pulses: - ModuleCommons (Enabled/Forced)
                                    - ModuleMelter (Enabled/Not Forced)
                                    - ModuleArmor (Enabled/Not Forced)
                                    - ModuleSteelworks (Enabled/Not Forced)
                                    - ChiselPlugin (Enabled/Not Forced)
                                    - Oredict (Enabled/Forced)
  Suspected Mods: Doomlike Dungeons (dldungeonsjbg)
  Profiler Position: N/A (disabled)
  Is Modded: Definitely; Server brand changed to 'fml,forge'
  Type: Dedicated Server (map_server.txt)

**Comment by BlackJar72** [comment]
It looks like there was something wrong with the block families files used to include more than one block in a wall of floor -- not really a code error but a configuration error (unless someone else modified the JSON reader which isn't likely and would be weird).

I can't really tell what is wrong without looking at the JSON files its trying to read.

Could you put this folder into a zip file and put it somewhere I can download it?

You can find it at...
[your Minecraft folder]/config/DLDungeonsJBG/BlockFamilies
...where "your Minecraft folder" is wherever the mods and configs are

I'm not an expert on JSON by any means, but I know the default block family files work for me.  If they've been changed for the mod pack or the server it was designed for there could be an error (or one of them could be corrupted).

Th contents of the files should look something like this, as far as format:

`{
  "name": "mixed_bricks_3", 
  "blocks": [ 
    "minecraft:stonebrick", 
    "minecraft:stonebrick(1)", 
    "minecraft:stonebrick(2)"  
  ]
} `

Basically, just the name of the family and a list of blocks it contains formated as modid:blockname(metadata).

I don't know if I can solve the problem, but I know I can't without looking at the file its trying to read when it throws the error to see if something looks off.

(Also, I have no idea what would happen if the block reference didn't exist, but that should be an error else where, not in reading the JSON.)

**Comment by LordBevis** [comment]
I can absolutely do that but would you happen to know a good easy to use place to upload it at? Or like what do you normally use for such things? Im really not sure how this error could have been created. I guess its possible something got corrupted during the data transfer process but it just seems so unlikely as it worked the last time me and my friend ran this server some years ago. I guess that dont mean alot because clearly something has gone wrong somehow. If this turns out to be the problem would you be able to give me the block family files that are properly working? I dont think the server data download is still up for this mod pack Im runnig or at least I cant see it on the download page on curseforge.

**Comment by BlackJar72** [comment]
I usually use MediaFire for most things, and I know there is a free version.  I know there are other options, but not what they all are or all the details.

**Comment by LordBevis** [comment]
Ok here is the block families folder as you had requested. Hope this sheds some light onto what's caused all this mess.
You can give me a non corrupted block families folder if I need it and you think that would fix it right? That would be super duper if you think its what is making all this happen that is. Thanks again for all the help with this.

https://www.mediafire.com/folder/uvkhfirjifpk5/BlockFamilies

**Comment by BlackJar72** [comment]
I see the problem.  There are a whole bunch of files that start with and underscore (and have the .json extension), like "_crumbling_walls1.json" which are not json files at all.

For example, the file listed above looks like this:

`
Mac OS X        2ATTR;com.apple.quarantineyZone.Identifier0081;00000000;;[ZoneTransfer]ZoneId=3ReferrerUrl=C:\Users\Mindcrack\Desktop\Hexxit+Updated-New+Year's+Edition+1.41+Server+Pack.zipThis resource fork intentionally left blank 
`

...but also contains a lot of non-printing characters that are not really valid for a text file (so its really a binary file).

I don't really know a lot about Mac OS, but to me this looks like an operating system specific soft link pointing to another file.  Was this moved between operating systems?  My first guess is that these linked to files somewhere else but the current operating system isn't understanding them so that when the mod looks for files in that folder it is given those as their raw contents?  Or maybe it went through a system that didn't recognize them and altered them?  Or maybe they are working but the files pointed to are not there?

I often use both "hard" and "soft" links on Linux -- and I know if I push something with a soft link to GitHub (accidentally forgetting there was such a file) as weird file is created in the repo.

**Comment by LordBevis** [comment]
Ah! that would make sense! The friend I got this from mostly runs stuff on macs so he probably ran the server that way.
Is there files or things you can give me to fix this problem? Do you perhaps have a windows iteration of server files? I could use new ones and just swap in the world data to then be able to run it. What do you think is gonna be the fix going forward? As I said before I dont see the server download files up for this any more on curseforge. Although I might be able to talk to the guy who made the pack and convince him to reupload them so I can use that to fix this problem. What do you think?

**Comment by BlackJar72** [comment]
If you just delete everything that starts with an underscore it should work.  If the linked files were modified versions, those would be gone, but it would remove the error and you'd still have all the default versions.

You might want to go through all the dungeon theme files, though, to look for any placed were the underscore name is listed as a block -- either remove those name or remove the underscore to use the default version.  Referencing a non-existent block family would cause a whole other error.

**Comment by LordBevis** [comment]
So I got rid of the two files called crumbling_walls1 and crumbling_walls2 in the block family folder as they were the ones that I could see that had underscores.
It still wont work it also did not auto regenerate those files in the block family folder.
Should I upload another crash log as it may be different from the one before or what do you think?
At this rate it may just make more sense for me to ask the modpack creator to reupload the server files so I can get new ones and move over the world data too it. This feels like its turning into open heart surgery. I know for a fact my friend ran this on MAC OS so thats probably the main cause of this whole mess but I dont wanna speak too soon if you think there is more that could be done to fix it.

**Comment by BlackJar72** [comment]
Perhaps the mod pack creator would have a better idea of what exactly those files are, being a Mac user and the one who added them.

If you removed those files, but files in the theme folder still reference them those block families it should still crash but a different -- I let it crash (fail-fast) rather than have a fail safe so that mod pack authors can find errors in their packs before releasing them.

You don't need to get rid of the normal versions of the files, just all the files with the actual underscore.  What you sent me was this:

![LBJsonFolder](https://github.com/user-attachments/assets/8bea423b-9754-4328-b616-6f5d4fbb3c4d)

Only the first half of those need to be deleted -- everything with an underscore on them, but not crumbling_walls1.json or crumbling_walls2.json, only the version with the underscore.

Alternately, you could remove the whole config folder for the mod, so it thinks its just been installed, and then it should regenerated everything.  I won't do this if you just remove a few file -- some people might want to just not use them after all.   This would remove any changes made for the mod pack, though; only vanilla blocks / mobs / items would be used (though a lot of mod packs don't change them anyway).

**Comment by LordBevis** [comment]
For the record the mod pack creator and my friend who ran this server are two different people. Before the server data download was removed Im sure it could also run on windows. its just how it generated the files differs from OS to OS.
Anyways we got a problem here lol check this out.

![data problem](https://github.com/user-attachments/assets/ea348eaa-69f1-4c63-b820-8d749af33430)

This is all I see in the folder which is different from yours. Its like Im missing a bunch of files. Id wager they are MAC OS files that my windows cant even see. This may be a bigger problem than I thought. I messaged the modpack creator about the server data files being reuploaded but I've not heard anything from them yet. I removed the whole block families folder. That did not work but it regenerated that folder and the files within it. or at least the ones that I could see.

**Comment by BlackJar72** [comment]
I meant to remove the entire DLDungeonsJBG folder, or just re-install the mod.  That should fix it, but it remove any changes (so any modded blocks / mob / loot that might have been added for the mod pack).

The mod pack creator might be better able to help though, as they would know what changes were made to the config.

**Comment by LordBevis** [comment]
I tried that and it did not work either. I think too many other files may be working thinking they are still on MAC OS.
Do you want me to upload the crash log for this last try or what do you have in mind to try next?

**Comment by BlackJar72** [comment]
If re-installing the mod didn't fix it, when it works outside the modpack and apparently worked in the modpack when run on Mac it probably isn't really a bug in the mod.  Perhaps either the modpack creator or your friend could help more.

I'd probably need to whole pack to test, but that would be on Linux not Windows, and I'm sorry I don't have time to debug the modpack for you.

**Comment by LordBevis** [comment]
Totally understandable! you have worked very hard up till now and your help is above and beyond! Ill try to see if the pack creator can help me out im sure that would fix it if i can get the files from him. I may wind up back here but it seems like we have just about squeezed this for all its juice. Ill try to work stuff out on my end from here on out if i can. Thanks for all the help dood!
